component-runtime-manager/src/main/java/org/talend/sdk/component/runtime/manager/extension/ComponentSchemaEnricher.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (C) 2006-2025 Talend Inc. - www.talend.com
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.talend.sdk.component.runtime.manager.extension;
+
+import static java.util.Collections.emptyMap;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Type;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Stream;
+
+import org.talend.sdk.component.api.input.Emitter;
+import org.talend.sdk.component.api.input.PartitionMapper;
+import org.talend.sdk.component.api.processor.Processor;
+import org.talend.sdk.component.api.service.schema.FixedSchema;
+import org.talend.sdk.component.api.standalone.DriverRunner;
+import org.talend.sdk.component.spi.component.ComponentMetadataEnricher;
+
+public class ComponentSchemaEnricher implements ComponentMetadataEnricher {
+
+    public static final String FIXED_SCHEMA_META_PREFIX = "tcomp::ui::schema::fixed";
+
+    public static final String FIXED_SCHEMA_FLOWS_META_PREFIX = "tcomp::ui::schema::flows::fixed";
+
+    public static final String SCHEMA_FLOW_DEFAULT = "__default__";
```

Comment:
Maybe could be define in [Output.java](https://github.com/Talend/component-runtime/blob/da88e5872597d1dd354d41a468b285b70ba1b3b8/component-api/src/main/java/org/talend/sdk/component/api/processor/Output.java#L26) as constant ?